frontend/src/components/App/Footer.jsx

```diff
@@ -23,31 +23,32 @@ const styles = {
 class Footer extends PureComponent {
 
     render() {
-        let track = {
-            img: '',
-            name: '',
-            artists: []
-        };
-        if (this.props.activePlayer !== null && this.props.activePlayer !== undefined) {
-
-            if (this.props.activePlayer.track !== null && this.props.activePlayer.track !== undefined) {
-                track = {
-                    img: this.props.activePlayer.track.album.images[this.props.activePlayer.track.album.images.length - 1].url,
-                    name: this.props.activePlayer.track.name,
-                    artists: this.props.activePlayer.track.artists
-                };
-            }
+        let activePlayer = this.props.activePlayer;
```

Comment:
We can destructure this object here

```suggestion
        let { activePlayer } = this.props;
```